mkt/search/tests/test_views.py

```diff
@@ -1389,6 +1389,12 @@ def test_search_preferred_region_match(self):
         objs = res.json['objects']
         eq_(len(objs), 0)
 
+    def test_region_explicit_none(self):
+        res = self.anon.get(self.url, data={'region': 'None'})
```

Comment:
It's really weird that the querystring parameter `None` is turned into the Python `None` value. Where does that happen?